schema/DollarName.hx

```diff
@@ -0,0 +1,11 @@
+#if (haxe_ver < 4.0)
```

Comment:
Wtf? `enum abstract` syntax is meant to look nicer, conditionalizing it for backwards-compatibility does the exact opposite and makes it look uglier instead...

Reply:
well, we need different values, so either I conditionalise it's fields, individual values or the full thing. I decided to go for a full replacement, because it is the least ugly variant.

Reply:
Hm.. not sure I agree, I actually completely missed that because the section that's conditionalized is much larger than it needs to be. :)